experiment/migrator/migrator.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python2
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Migrates information from Testgrid's Config.yaml to a subdirectory of Prow Jobs
+
+Only moves Dashboard Tabs
+Skips any Dashboard Tab Configuration that contains unusual keys, even if they're incorrect keys.
+"""
+
+import re
+from os import walk
+import ruamel.yaml
+
+PROW_DIR = "../../config/jobs"
+TESTGRID_CONFIG = "../../testgrid/config.yaml"
+
+def main():
+    confFile = open(TESTGRID_CONFIG, "r+")
+    config = ruamel.yaml.load(confFile, Loader=ruamel.yaml.RoundTripLoader, preserve_quotes=True)
+
+    for dashboard in config["dashboards"]:
+        dashNameA = dashboard["name"]
+        if "dashboard_tab" in dashboard:
+            for dashTab in dashboard["dashboard_tab"][:]:
+                if assertTabKeys(dashTab):
+                    moveTab(dashboard, dashTab, dashNameA)
+
+    confFile.seek(0)
+    ruamel.yaml.dump(config, confFile, Dumper=ruamel.yaml.RoundTripDumper)
+    confFile.truncate()
+    confFile.close()
+
+
+def moveTab(dashboard, dashTab, dashboardName):
```

Comment:
nit: prefer snake_case for python, eg: `move_tab`

Reply:
is `dashboardName` needed? we could get it from `dashboard["name"]`

Reply:
meta nit: mixing IO and logic in here makes this difficult to test in isolation, consider extracting reading and writing of prowjob config from this, such that this function just becomes logic